source/production/F0.Minesweeper.Components/Abstractions/MinefieldSize.cs

```diff
@@ -0,0 +1,6 @@
+namespace F0.Minesweeper.Components.Abstractions
+{
+	public sealed record MinefieldSize(uint Height, uint Width)
+    {
```

Comment:
_spaces_ here, rather than _tabs_

Reply:
Is github always showing the old state of the code when your comment was made? Because it should be fixed and it does look fixed in [the code file](https://github.com/Flash0ver/F0.Minesweeper/blob/feat/minefield-component/source/production/F0.Minesweeper.Components/Abstractions/MinefieldSize.cs) but this here makes me unsure 😅

Reply:
Yes - to keep the comment aligned with the code that was commented in the history of the PR.
It then labels the commented code with `Outdated` next to the full file name.